pom.xml

```diff
@@ -0,0 +1,153 @@
+    <!-- Compile -->
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>${commons-codec-version}</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>${commons-io.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.sonarsource.scanner.maven</groupId>
+      <artifactId>sonar-maven-plugin</artifactId>
+      <version>${sonar-maven-plugin.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>uk.gov.companieshouse</groupId>
+      <artifactId>structured-logging</artifactId>
+      <version>${structured-logging.version}</version>
+    </dependency>
+
+    <!-- Test -->
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>${junit-jupiter-engine.version}</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <profiles>
+    <profile>
+      <id>release-publish</id>
+      <activation>
+        <property>
+          <name>publishRepo</name>
+          <value>release</value>
+        </property>
+      </activation>
+      <properties>
+        <artifactoryReleasePublishRepo>libs-release-local</artifactoryReleasePublishRepo>
+      </properties>
+
+    </profile>
+    <profile>
+      <id>dev-publish</id>
+      <activation>
+        <property>
+          <name>publishRepo</name>
+          <value>dev</value>
+        </property>
+      </activation>
+      <properties>
+        <artifactoryPublishRepo>libs-snapshot-local</artifactoryPublishRepo>
+      </properties>
+    </profile>
+  </profiles>
+
+  <build>
+    <finalName>company-accounts-library</finalName>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>${maven-compiler-plugin.version}</version>
+        <configuration>
+          <source>${source-level}</source>
+          <target>${target-level}</target>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>${jacoco-maven-plugin.version}</version>
+        <executions>
+          <execution>
+            <id>default-prepare-agent</id>
+            <goals>
+              <goal>prepare-agent</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>default-report</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>report</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+
+  <repositories>
+    <repository>
+      <id>ch-artifactory</id>
+      <url>${artifactoryBaseUrl}/${artifactoryResolveRepo}</url>
+    </repository>
+  </repositories>
+
+  <distributionManagement>
```

Comment:
This bit can go into the dev publish and release publish profile as there are the only ones concerned with the distribution management of the app